protocol-designer/src/components/modules/MultipleModuleRow.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useDispatch } from 'react-redux'
+import {
+  LabeledValue,
+  OutlineButton,
+  ModuleIcon,
+  C_DARK_GRAY,
+  SIZE_1,
+  SPACING,
+} from '@opentrons/components'
+import { actions as stepFormActions, ModuleOnDeck } from '../../step-forms'
+import { DEFAULT_MODEL_FOR_MODULE_TYPE } from '../../constants'
+import { ModuleDiagram } from './ModuleDiagram'
+import { FlexSlotMap } from './FlexSlotMap'
+import type { ModuleModel, ModuleType } from '@opentrons/shared-data'
+
+import styles from './styles.module.css'
+
+interface MultipleModuleRowProps {
+  type: ModuleType
+  openEditModuleModal: (moduleType: ModuleType, moduleId?: string) => void
+  moduleOnDeckType?: ModuleType
+  moduleOnDeckModel?: ModuleModel
+  moduleOnDeck?: ModuleOnDeck[]
+}
+
+export function MultipleModuleRow(props: MultipleModuleRowProps): JSX.Element {
+  const {
+    moduleOnDeck,
+    openEditModuleModal,
+    moduleOnDeckModel,
+    moduleOnDeckType,
+    type: propsType,
```

Comment:
from interface, maybe moduleType is easy to understand?

```suggestion
    type: moduleType,
```